src/util/__tests__/createServerRootMixin.test.js

```diff
@@ -262,6 +263,64 @@ Array [
 
       await renderToString(wrapper);
     });
+
+    it('forwards vuex', async () => {
+      const searchClient = createFakeClient();
+
+      Vue.use(Vuex);
+
+      const store = new Vuex.Store({
+        state: {
+          count: 0,
+        },
+        mutations: {
+          increment(state) {
```

Comment:
We don't use this in the test. Do we need it?

Reply:
ah yes, good point! I copied from my experiments earlier, but we don't need a full fledged store